.gitignore

```diff
@@ -3,3 +3,4 @@ eTaxInvoicePdfGenerator/bin/
 /eTaxInvoicePdfGenerator.csproj.user
 eTaxInvoicePdfGenerator/obj/
 eTaxInvoicePdfGenerator/publish/
+packages/
```

Comment:
Add nuget package temporary folder to .gitignore. So we're not going to add unnecessary binary to repository.